docs/nodes/widgets/ui-switch.md

```diff
@@ -5,8 +5,8 @@ props:
     Label: The text shown within the button.
     On Payload: The type & value to output in <code>msg.payload</code> when the switch is turned on.
     Off Payload: The type & value to output in <code>msg.payload</code> when the switch is turned off.
-    On Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "on" state
-    Off Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "off" state
+    On Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "on" state. No need to include the <code>mdi</code> prefix.
+    Off Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "off" state. No need to include the <code>mdi</code> prefix.
```

Comment:
I understand the need for clarity but should we not also support mdi (as in `mdi-copy`)?

Just thinking of a future where we permit `fa-copy` and `fas-copy`?

Reply:
We do already support that - but just pointing out it's not required

Reply:
The `mdi` library is far larger than `fa` anyway, so I don't think we'll need to venture away anytime soon

Reply:
understood. So copy will use `mdi-copy` and (if sometime fown the road we support fas, that would be fas-copy)

Reply:
yep